src/index.html

```diff
@@ -4,9 +4,329 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+          class="header__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+            <!-- <a href="#" class="nav__link is-active">
+              Apple
+            </a> -->
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="catalog__card-img"
+        >
+        <h2 class="catalog__card-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <section class="catalog__card-product-code">Product code: 195434</section>
+        <section class="catalog__card-stars-and-reviews">
+
+          <div class="catalog__card-stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+
+          <div class="catalog__card-reviews">Reviews: 5</div>
+
+        </section>
+
+        <section class="catalog__card-price">
+          <p class="catalog__card-price-name">Price:</p>
+          <p class="catalog__card-price-value">$2,199</p>
+        </section>
+
+        <a
+          href="#"
+          class="catalog__card-button"
+          data-qa="card-hover"
+        >
+        Buy
+        </a>
+      </div>
+      <div class="catalog__card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="catalog__card">
```